### DISCUSSION TITLE
Phantom mode question

### DISCUSSION THREAD

**Discussion by ManuelSvQ** [winstxnhdw/lc-hax#161] [Q&A]
Is there any way so you can disappear fully from other players whilst in phantom mode? I've been called out for "being afk" whilst being in phantom mode  and I need to know if this is possible.

**Comment by winstxnhdw** [comment]
So what I do is use `FollowMod` in combination with `Phantom`. Another thing you can do is go hide in some area and do `/block radar`.

**Reply by ManuelSvQ** [reply]
I haven't been able to know how follow player works, as it doesn't seem to do anything.

**Reply by winstxnhdw** [reply]
M3 + F while looking at a person

**Reply by yujispin990** [reply]
btw the followmod is one of the most insane thing i've ever seen holy moly!

**Reply by yujispin990** [reply]
if you follow a person while outside the ship and the ship is leaving, you are going to fly towards the ship following the person XD.

**Reply by winstxnhdw** [reply]
Wait, how are you guys only finding out about this now 😭 It's one of the oldest mods..

**Reply by yujispin990** [reply]
it replicates movements PERFECTLY, even when a person falls it automatically stops following, beautiful work @winstxnhdw !!!

**Reply by yujispin990** [reply]
i've been using it for awhile, but i didnt give proper praises for how insane the mod really is, seriously it is amazing!

**Reply by winstxnhdw** [reply]
Yeap, honestly beautiful work from @LTYGUY.

**Reply by yujispin990** [reply]
oh he made it? thank you so much @LTYGUY !!!

**Reply by winstxnhdw** [reply]
Could someone help me test `/invis`? It uses a very similar concept to the `FollowMod`.

**Reply by joep26020** [reply]
does block radar block spectators too?

**Reply by joep26020** [reply]
what if somebody tries to spectate you with invis on?

**Reply by winstxnhdw** [reply]
No way to block spectators. I just do `/invis` and people think my game is bugged.

**Reply by winstxnhdw** [reply]
> what if somebody tries to spectate you with invis on?

They'll just see you floating in oblivion. Most people don't care.

**Reply by joep26020** [reply]
K thanks
